README.md

```diff
@@ -29,18 +29,18 @@ Current image in Docker Hub:  `openliberty/operator:0.0.1`
   * `kubectl apply -f deploy/role_binding.yaml`
   * `kubectl apply -f deploy/operator.yaml`
 
-* Install SCC
-  * `cd ..`
-  * `git clone https://github.com/IBM/charts.git` 
-    * Should be cloned to the same directory that contains the cloned open-liberty-operator
-  * `cd charts/stable/ibm-open-liberty/ibm_cloud_pak/pak_extensions/pre-install/clusterAdministration`
-  * `./createSecurityClusterPrereqs.sh`
-  * `cd ../namespaceAdministration`
-  * `./createSecurityNamespacePrereqs.sh <namespace>`
+* Install security
+  * For OpenShift install SCC
+    * `kubectl apply -f deploy/ibm-open-liberty-scc.yaml --validate=false`
+    * `oc adm policy add-scc-to-group ibm-open-liberty-scc system:serviceaccounts:<namespace>`
+      * `Update <namespace> with the appropriate namespace`
+  * `kubectl apply -f deploy/ibm-open-liberty-psp.yaml`
```

Comment:
PSP (Pod Security Policy) wouldn't apply to OpenShift, as that is the equivalent of SCC for non-OpenShift envs.

Reply:
So should I say for non-OpenShift set up Pod Security Policy.   Reason I put both is because the script would do both if the both sets of Kubernetes resource name/Kubernetes API version were found. When I issued the kubectl commands in my OpenShift environment the Kubernetes resource name/Kubernetes API version for PSP were found.

Reply:
Tim and I had a sort-of-lengthy discussion about this yesterday.  The script seems to set either or both, if the underlying kube implementation understands what that type is.  Since OpenShift knows what a PSP is, the script will create one.  Out of the box, OpenShift won't use a PSP, even though it understands what it is.  But, you can turn on the PSP admission controller in OpenShift and then all of a sudden it starts using them.  I'm not advocating for keeping the instructions the way that they are, maybe I'm just making sure that we all understand what is really going on here, and that the scripts we ship with Helm are really not what we think they are either.  I think the instructions should say OpenShift gets the SCC, and ICP gets the PSP, and if you're not using either of those you have to infer what you might need based on how you set up kube.

Reply:
I just pushed a commit that adds a header "For IBM Cloud Private install PSP" above the psp stuff.

Reply:
Arthur and I spoke about this, there was a mis-understanding about whether we'd have the yaml files for the SCC and PSP in this git repository.  We're all on the same page now.  You can merge.